docs/team.rst

```diff
@@ -79,9 +79,12 @@ Roles
 Members
 ```````
 
-* `Mike Jang <https://twitter.com/TheMikeJang>`__, Senior Staff Technical Writer, ForgeRock *
-* `Margaret Eker <https://twitter.com/meker>`_
-* `Rose Williams <https://twitter.com/ZelWms>`_
+* `Rose Williams <https://twitter.com/ZelWms>`_ *
+* `Margaret Eker <https://twitter.com/meker>`_ *
+* `Mike Jang <https://twitter.com/TheMikeJang>`_ *
+* `Marybeth Alexander <https://twitter.com/knowledge_owl>`_
+* `Catherine Heath <https://twitter.com/awaywithwords2>`_
+* `Swapnil Ogale <https://twitter.com/swapnilogale>`_
```

Comment:
I wonder if we should split this up into regions on this page? So people know who to contact about which region (UTC-8, UTC, UTC+8 unless we have a better grouping?)

Reply:
Sounds reasonable.